src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from "classnames";
+import Icon, { ICONS } from "../Icon";
+import React, { HTMLProps, ReactNode, useRef, useState } from "react";
```

Comment:
Build/lint seems to fail in this. 
```suggestion
import React, { HTMLProps, useRef, useState } from "react";
```